### DISCUSSION TITLE
Support for Whisper API?

### DISCUSSION THREAD

**Discussion by ffalcon31415** [HenestrosaDev/audiotext#42] [Ideas]
I'd love to have support for openai's Whisper API. My computer too slow to do the work and Whisper's API results far better than google's in my experience.

Thanks for the work on this tool!

**Comment by HenestrosaDev** [comment]
This is definitely an interesting implementation that I'd be happy to add. I'll put it on the Roadmap and start working on it when I have the time.

Thanks for the suggestion!

**Comment by HenestrosaDev** [comment]
Made in #45. Will be available in version 2.3.0.

**Comment by HenestrosaDev** [comment]
Version 2.3.0 released. You can now use Whisper API in Audiotext!
